controller/src/main/java/org/jboss/as/controller/access/constraint/SensitiveTargetConstraint.java

```diff
@@ -149,5 +155,10 @@ protected int internalCompare(AbstractConstraintFactory other) {
             // We have no preference
             return 0;
         }
+
+        @Override
+        public Constraint getRequiredConstraint(Action.ActionEffect actionEffect, JmxAction action, JmxTarget target) {
+            return (action.getImpact() == JmxAction.Impact.EXTRA_SENSITIVE || target.isNonFacadeMBeansSensitive() || isSensitiveAction(action.composeAction(), actionEffect) || isSensitiveResource(target, actionEffect)) ? SENSITIVE : NOT_SENSITIVE;
```

Comment:
isSensitiveAction() and isSensitiveResource() will always return false, so the calls can be dropped.